src/components/Place/Place.js

```diff
@@ -0,0 +1,32 @@
+import React, {useEffect, useState} from 'react';
+import { PlaceContainer } from './PlaceComponents';
+
+
+const Place = () => {
+
+const [apiData, setApiData] = useState([])
+
+const getData = () => {
+    fetch('./API/database.json')
+    .then(response => {
+        if(response.ok) return response
+        throw new Error(response.status)
+    })
+    .then(response => response.json())
+    .then(data => {
+        setApiData(data);
+    })
+ }
+
+useEffect(() => {
+    getData();
+    return () => {
+    setApiData([]);
+    }
+}, [])
+
+
+    return <><PlaceContainer apiData={apiData}/></>
```

Comment:
React.Fragment isn't necessary here.

Reply:
As @weronikaolejniczak mentioned. You can remove a React.Fragment in return.